packages/analytics-client-common/src/default-tracking.ts

```diff
@@ -108,3 +93,53 @@ export const getPageViewTrackingConfig = (config: BrowserOptions): PageTrackingO
     eventType,
   };
 };
+
+export const isAttributionTrackingEnabled = (defaultTracking: DefaultTrackingOptions | boolean | undefined) => {
```

Comment:
note: adops shorthand config for attribution, similar to other default tracking configs